chips/sifive/src/prci.rs

```diff
@@ -48,29 +51,111 @@ register_bitfields![u32,
 
 pub enum ClockFrequency {
     Freq16Mhz,
+    Freq320Mhz,
 }
```

Comment:
What would be the best way to support more frequencies? Is it possible/do we want to support arbitrary frequencies (well, semi-arbitrary, there are rules about what frequencies are valid).

Reply:
I think an enum of supported and tested frequencies is the best option

Reply:
Clocks are complicated, but the reality is most people just use the same crystals and oscillators that others use, and we may never see additional boards using the sifive g0002 chip. I wouldn't worry about trying to support what is possible, only what we actually need to use.